codebase/config/sync/migrate_plus.migration.idc_ingest_new_items.yml

```diff
@@ -138,154 +188,191 @@ process:
+            plugin: entity_lookup
             entity_type: taxonomy_term
             bundle: person
             bundle_key: vid
             value_key: name
   field_date_available: date_available
   field_date_copyrighted:
-    - plugin: skip_on_empty
+    -
+      plugin: skip_on_empty
       method: process
       source: date_copyrighted
-    - plugin: explode
+    -
+      plugin: explode
       delimiter: '|'
       strict: false
   field_date_created:
-    - plugin: skip_on_empty
+    -
+      plugin: skip_on_empty
       method: process
       source: date_created
-    - plugin: explode
+    -
+      plugin: explode
       delimiter: '|'
       strict: false
   field_date_published:
-    - plugin: skip_on_empty
+    -
+      plugin: skip_on_empty
       method: process
       source: date_published
-    - plugin: explode
+    -
+      plugin: explode
       delimiter: '|'
       strict: false
   field_description:
-    - plugin: skip_on_empty
+    -
+      plugin: skip_on_empty
       method: process
       source: description
-    - plugin: explode
+    -
+      plugin: explode
       delimiter: '|'
       strict: false
-    - plugin: deepen
-    - plugin: sub_process
+    -
+      plugin: deepen
+    -
+      plugin: sub_process
       process:
         value:
-          - plugin: skip_on_empty
+          -
+            plugin: skip_on_empty
             method: process
             source: value
-          - plugin: explode
-            delimiter: ';'
-          - plugin: extract
+          -
+            plugin: explode
+            delimiter: ;
+          -
+            plugin: extract
             index:
               - 0
         target_id:
-          - plugin: skip_on_empty
+          -
+            plugin: skip_on_empty
             method: process
             source: value
-          - plugin: explode
-            delimiter: ';'
-          - plugin: extract
+          -
+            plugin: explode
+            delimiter: ;
+          -
+            plugin: extract
             index:
               - 1
-          - plugin: entity_lookup
+          -
+            plugin: entity_lookup
             entity_type: taxonomy_term
             bundle: language
             bundle_key: vid
             value_key: field_language_code
   field_digital_identifier:
-    - plugin: skip_on_empty
+    -
+      plugin: skip_on_empty
       method: process
       source: digital_identifier
-    - plugin: explode
+    -
+      plugin: explode
       delimiter: '|'
       strict: false
   field_digital_publisher:
-    - plugin: skip_on_empty
+    -
+      plugin: skip_on_empty
       method: process
       source: digital_publisher
-    - plugin: explode
+    -
+      plugin: explode
       delimiter: '|'
       strict: false
-    - plugin: entity_lookup
+    -
+      plugin: entity_lookup
       entity_type: taxonomy_term
       bundle: corporate_body
       bundle_key: vid
       value_key: name
-  # note: only available in JSON view
```

Comment:
Removed comment due to formatting?

Reply:
Same as above -- drupal reformats the file on a config import/export.

Reply:
Resolving!